src/test/java/com/example/survey/models/QuestionTest.java

```diff
@@ -22,4 +29,49 @@ void testEquals() {
         question2.setId(3);
         assertTrue(question.equals(question2));
     }
+
+    @Test
+    void testTextQuestionJsonSerialization() {
+        question.id = 3;
+        question.addAnswer(new Answer("Joe"));
+        question.addAnswer(new Answer("Jim"));
+        try {
+            String result = new ObjectMapper().writeValueAsString(question);
+            assertThat(result, containsString("\"question\":\"What is your name?\""));
```

Comment:
I think you should be able to use `jsonPath` here instead, similar to the survey controller tests

Reply:
^ applies to all tests that have an expected json value

Reply:
The same package can't be used. I tried importing [JsonPath](https://github.com/json-path/JsonPath) instead, but I could not get that working here.